scripts/main.js

```diff
@@ -0,0 +1,337 @@
+//Initialization of DOM variables
+//Buttons initialization
+let start = document.getElementById("start");
+let reset = document.getElementById("reset");
+
+//status elements initialization 
+let statusBar = document.getElementById("statusBar");
+let gameTitle = document.getElementById("game-title");
+let cells = Array.from(document.getElementsByClassName("cell"));
+
+//will store our current game here, empty array helps to track the played cells.
+let boxes = ["", "", "", "", "", "", "", "", ""];
+let gameStatus = true;
+let totalTurns = 0;
+
+// Initialization of player variables
+let currentPlayer = "X";
+let player1 = document.getElementById("player1");
+let player2 = document.getElementById("player2");
+let playerVsPlayer = document.getElementById("playerVPlayer");
+let playerVsComputer = document.getElementById("playerVComputer");
+let gameType = "";
+
+
+//Clock timer Initialization
+let hours = document.getElementById("hours");
+let minutes = document.getElementById("minutes");
+let seconds = document.getElementById("seconds");
+let totalSeconds = 0;
+
+//----Check pVp or pVComp-----//
+// start is disabled until player selects the game type:pVp or pVcomp
+start.disabled = true;
+reset.disabled = true;
+playerVsComputer.addEventListener("click", () => {
+  player1.disabled = true;
+  player2.disabled = true;
+  playerVsPlayer.disabled = true;
+  playerVsComputer.disabled = true;
+  start.disabled = false;
+  reset.disabled = false;
+  gameType = "pVComp";
+  currentPlayer = "X";
+  statusBar.textContent = "Click Start Button";
+});
+
+playerVsPlayer.addEventListener("click", () => {
+  
+  playerVsPlayer.disabled = true;
+  playerVsComputer.disabled = true;
+  start.disabled = false;
+  reset.disabled = false;
+  gameType = "pVp";
+  currentPlayer = "X";
+  statusBar.textContent = "Click Start Button";
+});
+
+//-----Function to decide players turn---->
+//----Start button function----//
+//Start button fading out and displaying player
+
+start.addEventListener("click", startGame);
+
+function startGame() {
+  start.disabled = true;
+  totalTurns = 0;
+  // disables the button once clicked
+  statusBar.textContent = "Enjoy the Game";
+  gameTimer = setInterval(setTime, 1000, true);
+  if (gameType === "pVp") {
+    playerGame()
+  } else if (gameType === "pVComp") {
+    computerGame()
+  }
+}
```

Comment:
Breaking up your game functions into smaller more manageable pieces and then calling those functions/pieces when needed was a smart way to go about attacking this project. Breaking things up into smaller chunks will be huge when working on more complex projects.